### DISCUSSION TITLE
Download

### DISCUSSION THREAD

**Discussion by Incephrer** [nesbox/TIC-80#1715] [General]
For some versions of Linux, the install process does not work. I think that there should be an option for downloading an INSTALL.sh file for installing the program using the terminal.

**Comment by paul59** [comment]
> For some versions of Linux, the install process does not work. I think that there should be an option for downloading an INSTALL.sh file for installing the program using the terminal.

Do you mean 'build' rather than install (since no 'installation' is required)?
